src/uuid.cr

```diff
@@ -157,12 +157,7 @@ struct UUID
     @bytes.to_unsafe
   end
 
-  # Returns `true` if `other` string represents the same UUID, `false` otherwise.
-  def ==(other : String)
```

Comment:
Why did you remove this?

Reply:
I said exactly my reasoning above, it's magic, and costly, and unexpected.